flink-runtime/src/main/java/org/apache/flink/runtime/metrics/MetricNames.java

```diff
@@ -49,6 +54,27 @@ private MetricNames() {
 	public static final String IO_CURRENT_INPUT_WATERMARK_PATERN = "currentInput%dWatermark";
 	public static final String IO_CURRENT_OUTPUT_WATERMARK = "currentOutputWatermark";
 
+	private static final String  SHUFFLE_NETTY_GROUP = new StringJoiner(OperatorScopeFormat.SCOPE_SEPARATOR)
```

Comment:
Suggested to simplify it by writing this way: `java.lang.String#join(java.lang.CharSequence, java.lang.CharSequence...)`